### ISSUE TITLE
Setting 2 multivariate flag states with 50/50 weight prevents overrides 

### ISSUE THREAD

**Issue by dabeeeenster** [Flagsmith/flagsmith#234]
Steps to reproduce:

1. Create new flag.
2. Create 2 variations, each one with 50% weight.
3. Store flag. 
4. Open flag, go to overrides
5. Select any segment
6. Select one of the 50% variants
7. Try and store flag

API returns a 500 with the following logged:

```
['Total percentage allocation for feature highly-segmented-feature in environment Development is greater than 100%.']
```

**Comment by dabeeeenster** [comment]
Simpler reproduction:

1. Create feature with 2 variants 50 / 50
2. Save
3. Open and update flag (without changing anything)

**Comment by dabeeeenster** [comment]
Looks like https://github.com/Flagsmith/flagsmith/blob/3c2814c0d13cadcacca61c2c9e2bea5b1bb5d560/api/features/multivariate/models.py#L60 is incorrect

**Comment by matthewelwell** [comment]
Caused by the fact that the FE is not sending the id of each of the `multivariate_feature_state_values` in the `PUT` request so the BE thinks it's trying to create new ones and hence it was failing trying to create a third multivariate feature state, pushing the total percentage to 150%. 

Example PUT request:

```json
{
    "id": 59285,
    "feature_state_value": "big",
    "multivariate_feature_state_values": [
        {
            "multivariate_feature_option": 325,
            "percentage_allocation": 50
        },
        {
            "multivariate_feature_option": 326,
            "percentage_allocation": 50
        }
    ],
    "identity": null,
    "enabled": false,
    "feature": 11215,
    "environment": 7478,
    "feature_segment": null
}
```

As you can see, none of the `multivariate_feature_state_values` have an `id` attribute (although it is returned on the GET request). Setting this value will ensure that the API is aware this is NOT a new value and hence will not try to create a new feature state value. 

I will fix up the BE validation to ensure that multiple multivariate feature state values cannot be created for the same multivariate feature option, however, this will just change the error message that is thrown so the FE fix is the key part.

**Comment by kyle-ssg** [comment]
https://github.com/Flagsmith/flagsmith/pull/236

**Comment by dabeeeenster** [comment]
Fixed in #236
